src/System/ref/Compat/System.cs

```diff
@@ -4995,14 +4995,14 @@ public partial class CookieCollection : System.Collections.ICollection, System.C
         public CookieCollection() { }
         public int Count { get { return default(int); } }
         public bool IsReadOnly { get { return default(bool); } }
-        public bool IsSynchronized { get { return default(bool); } }
+        bool System.Collections.ICollection.IsSynchronized { get { return default(bool); } }
+        object System.Collections.ICollection.SyncRoot { get { return default(object); } }
```

Comment:
>These are explicit interface implementation methods so they cannot be public

So, that means that they are "private"?  I think the access modifier should be explicitly listed "public", "protected", "private", etc.  That is per CoreFx coding guidelines.